Wetstone.csproj

```diff
@@ -55,8 +53,8 @@
       <HintPath>$(UnhollowedDllPath)\com.stunlock.network.dll</HintPath>
     </Reference>
 
-    <Reference Include="UnityEngine.InputLegacyModule.dll">
-      <HintPath>$(UnhollowedDllPath)\UnityEngine.InputLegacyModule.dll</HintPath>
+    <Reference Include="Stunlock.Localization">
+      <HintPath>..\BepInEx\unhollowed\Stunlock.Localization.dll</HintPath>
```

Comment:
I'd prefer to use `<PackageReference Include="VRising.Unhollowed.Client" Version="0.6.0.57108*" />` over local refs.

Reply:
That's not updated for the current hotfix patch number. Also, I didn't want to change the project, just update it. I guess I can do that though.

Reply:
Thanks for the reminder I've updated it on nuget.org: `<PackageReference Include="VRising.Unhollowed.Client" Version="0.6.1.*" />`

Reply:
This doesn't need to be in this PR, though I'd love for it to have that validation even right now, it can be done separately.